### PR TITLE
COM hijack & UAC bypass via Sdclt

### PR DESCRIPTION
**Pull request by omkar72**
Adding shell folder registry key with application to execute can bypass UAC and work in high integrity context. Rule will monitor creation of registry key

### CODE REVIEW

**Comment by omkar72** [comment]
Raising a new pull request with updated commit

File: rules/windows/registry_event/sysmon_comhijack_sdclt.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - attack.t1546
+    - attack.t1548
+author: Omkar Gudhate
+date: 2020/09/27
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    selection:
+        TargetObject:
```
Comment (Neo23x0) [inline rules/windows/registry_event/sysmon_comhijack_sdclt.yml:19]: I don't understand the TargetObject selection. 
Isn't a registry access to this key pretty common `HKCU\Software\Classes\Folder\shell\open\command`?

Comment (omkar72) [reply rules/windows/registry_event/sysmon_comhijack_sdclt.yml:19]: Oh, combining folder\shell\command & value as ‘DelegateExecute’ I didnt see much false positives. This needs to be set in order to bypass uac through sdclt execution. Should i add sysmon id 13 in category ? Or add sdclt process execution condition as well?

Comment (omkar72) [reply rules/windows/registry_event/sysmon_comhijack_sdclt.yml:19]: Also i opened up a new duplicate pr for same logic. New to sigma rule creations, might need few more suggestions before getting it right

**Pull request closed without merging**